mrchecker-framework-modules/mrchecker-core-module/src/main/java/com/capgemini/mrchecker/test/core/base/environment/GetEnvironmentParam.java

```diff
@@ -1,31 +0,0 @@
-package com.capgemini.mrchecker.test.core.base.environment;
-
-import com.capgemini.mrchecker.test.core.BaseTest;
-import com.capgemini.mrchecker.test.core.exceptions.BFInputDataException;
-
-enum GetEnvironmentParam {
```

Comment:
Why it is deleted? I think it is not connected to Junit5mig, is it?

Reply:
This class was never used.